cfgov/v1/atomic_elements/organisms.py

```diff
@@ -854,7 +854,11 @@ class FilterableList(BaseExpandable):
         required=False,
         help_text='Additional explanation for the notification that '
                   'will be displayed if there are no posts to filter.')
-    post_date_description = blocks.CharBlock(default='Published')
+    post_date_description = blocks.CharBlock(
+        required=False, label='Date stamp descriptor',
```

Comment:
Trivial: There are a lot of older places where we're not consistent about this, but I think our preference is, when arguments must be broken onto multiple lines, to begin each argument on its own line.

Reply:
Sounds good 👍